build/nuget/Uno.WinUI.Skia.Gtk.nuspec

```diff
@@ -23,7 +23,5 @@
   <files>
 	<file src="..\..\src\Common\uno.png" target="/" />
 	<file src="_._" target="lib\netstandard1.0" />
-	<file src="Uno.WinUI.Skia.Gtk.props" target="build" />
-	<file src="Uno.WinUI.Skia.Gtk.targets" target="build" />
```

Comment:
We now can get rid of this as they are moved to `Uno.WinUI.Runtime.Skia.Gtk` package (brought as a dependency by Uno.WinUI.Skia.Gtk, see line 19)